plancheck.go

```diff
@@ -11,32 +11,47 @@ import (
 )
 
 func init() {
+	ignoreEmpty := false
```

Comment:
Vil ignore empty ikke være at fortrække som default

Reply:
I believe the default should be to include filesystems without snapshots.

The command is described as `Print mounts that have no plan and exit`, `--ignore-empty` will deviate from that.

Reply:
Ok. So change the check plan description to Print mounts that have snapshot but no plan to clean them and exit.
And change --ignore-empty to --include-empty
I belive that the majority of use cases would ignore empty.